### PR TITLE
Publicize scheduling: add stats

### PR DESCRIPTION
**Pull request by artpi**
This adds various stats for publicize scheduling usage.

- calypso_publicize_share_schedule for scheduling shares
- calypso_publicize_share_instantly for instant republicize action
- calypso_publicize_post_share_view for opening share component
- calypso_publicize_action_tab_click for clicking tabs with sheduled / publithed
- calypso_publicize_scheduled_delete for deleting a share
- calypso_publicize_share_preview_toggle for opening preview window

## Testing

Set logging in calypso:
```javascript
localStorage.setItem('debug', 'calypso:analytics:*');
```

Perform the above actions and see if stats come in console.

### CODE REVIEW

**Comment by retrofox** [comment]
Testing stats 

- [ ] calypso_publicize_share_schedule for scheduling shares
- [ ] calypso_publicize_share_instantly for instant republicize action
- [ ] calypso_publicize_post_share_view for opening share component
- [ ] calypso_publicize_action_tab_click for clicking tabs with sheduled / publithed
- [ ] calypso_publicize_scheduled_delete for deleting a share
- [ ] calypso_publicize_share_preview_toggle for opening preview window

**Comment by retrofox** [comment]
image

**Comment by retrofox** [comment]
image

**Comment by retrofox** [comment]
image

**Comment by retrofox** [comment]
image

File: client/my-sites/post-share/publicize-actions-list.jsx [hunk 1]
```diff
@@ -28,6 +28,7 @@ import NavItem from 'components/section-nav/item';
 import { isEnabled } from 'config';
 import Dialog from 'components/dialog';
 import { deletePostShareAction } from 'state/sharing/publicize/publicize-actions/actions';
+import analytics from 'lib/analytics';
 
 class PublicizeActionsList extends PureComponent {
```
Comment (retrofox) [inline client/my-sites/post-share/publicize-actions-list.jsx]: Sidenote. This component isn't Pure.

**Comment by retrofox** [comment]
Also, when a scheduled action is deleted it's like all `calypso_publicize_scheduled_delete` stats are triggered, one for each scheduled action.

**Comment by retrofox** [comment]
It's not possible test `calypso_publicize_share_preview_toggle` for now, right?

**Comment by artpi** [comment]
> It's not possible test calypso_publicize_share_preview_toggle for now, right?

It should be, press "Published" or press "Scheduled" after

> Also, when a scheduled action is deleted it's like all calypso_publicize_scheduled_delete stats are triggered, one for each scheduled action.

Nice catch, Ill fix that

**Comment by retrofox** [comment]
> > It's not possible test calypso_publicize_share_preview_toggle for now, right?
>
> It should be, press "Published" or press "Scheduled" after

I'm sorry I don't get it. Do you mean from actions list?

**Comment by artpi** [comment]
> Also, when a scheduled action is deleted it's like all calypso_publicize_scheduled_delete stats are triggered, one for each scheduled action.

fixed, nice catch!

> It's not possible test calypso_publicize_share_preview_toggle for now, right?

zrzut ekranu 2017-06-14 o 17 58 46

**Comment by retrofox** [comment]
It's weird I don't that button :-o

**Comment by jhnstn** [comment]
LGTM :shipit:

**Pull request merged**